internal/services/nginx/nginx_certificate_resource_test.go

```diff
@@ -196,6 +196,10 @@ resource "azurerm_virtual_network" "test" {
   address_space       = ["10.0.0.0/16"]
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
+
+  lifecycle {
```

Comment:
Hi @stephybun Can you please elaborate on why this was added to the NGINXaaS tests? I see that the subnet below is created for this test and should not causes any issues which will lead to a user requiring `ignore_changes` on the vnet.

Reply:
@puneetsarna this was added because of a behavioural change that we were planning to roll out in 4.0 which would require all virtual network resources to apply `ignore_changes` on `subnet` if the subnet is being managed outside of the virtual network resource. This scenario applies to the majority of our test cases, nginx ones included.

However given the huge impact this would have on users we've decided to roll back this change. We're working through this at the moment, so this will be reverted in the near future.